deploy/examples/create-external-cluster-resources.py

```diff
@@ -60,11 +61,9 @@
 
 py3k = False
 try:
-    from urlparse import urlparse
     from base64 import encodestring
```

Comment:
Don't remove this it's checking if it's pyhon2 or python3.

Don't remove it from here line #63 #67

Reply:
we are already doing try and except in above so why we need twice?